.gitignore

```diff
@@ -12,3 +12,5 @@ docs/man
 npm-debug.log
 .project
 test/memory.json
+intl/*
+!intl/en
```

Comment:
Now I added an empty line at the end of file; not sure why it does not show it here.